### PR TITLE
quivira: init at 4.1

### PR DESCRIPTION
**Pull request by nosewings**
###### Description of changes

Adds the Quivira font.

Note that this does not build on Darwin platforms.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/data/fonts/quivira/default.nix
Comment (RaitoBezarius) [inline pkgs/data/fonts/quivira/default.nix:23]: homepage

**Comment by RaitoBezarius** [comment]
Thank you for your contribution, sorry it took a while.

- Please, can you separate the maintainer changes into a separate commit?
- Please, can you fix the conflicts?
- Please, can you add comments on why recursive / downloadToTemp is needed?

**Comment by nosewings** [comment]
Issues resolved!

File: pkgs/data/fonts/quivira/default.nix
Comment (RaitoBezarius) [inline pkgs/data/fonts/quivira/default.nix:19]:
```suggestion
  meta = with lib; {
```

Comment (RaitoBezarius) [reply pkgs/data/fonts/quivira/default.nix:19]: This change makes it easier to write the next lines, this is not mandatory for this PR.

**Review by RaitoBezarius — APPROVED** [review]
LGTM, I will give it a bit of time and merge it if no one oppose it. :)

**Comment by RaitoBezarius** [comment]
Thank you for your contribution!

**Comment by RaitoBezarius** [comment]
Hi @nosewings ; apologies for the delay (I was away from keyboard for these last weeks), I think no one opposed, if you could fix the conflict, I think I will merge it just after, thank you again for your contribution!

**Review by emilytrau — APPROVED** [review]
Thank you for your contribution!

**Pull request merged**
